### PR TITLE
add arm image

### PR DESCRIPTION
**Pull request by lengrongfu**
issue https://github.com/goharbor/harbor-operator/issues/993

### CODE REVIEW

File: Dockerfile-multi-arch [hunk 1]
```diff
@@ -0,0 +1,15 @@
+FROM golang:1.18.3 as build
```
[the remark this answers is not in the document]
Comment (lengrongfu) [reply Dockerfile-multi-arch:1]: Replacing Dockerfile is ok.

File: .dockerignore [hunk 2]
```diff
@@ -17,8 +17,8 @@ node_modules
 *.out
 
 # Kubernetes Generated files - skip generated files, except for vendored files
-**/zz_generated.*
-!vendor/**/zz_generated.*
+# **/zz_generated.*
```
Comment (thcdrt) [inline .dockerignore:20]: Why commenting and not deleting those lines ?

File: Dockerfile [hunk 3]
```diff
@@ -1,8 +1,15 @@
+FROM golang:1.18.3 as build
+WORKDIR /workspace
+ENV GO111MODULE=on \
+    GOPROXY=https://goproxy.cn,direct
```
Comment (thcdrt) [inline Dockerfile:4]: What is GOPROXY used for ?

**Review by thcdrt — APPROVED** [review]
LGTM

File: .github/workflows/release.yml [hunk 4]
```diff
@@ -47,31 +47,38 @@ jobs:
...
+          password: '${{ secrets.GHCR_TOKEN }}'
+      - name: Build and push go ghcr
+        uses: docker/build-push-action@v3
+        with:
+          push: true
+          platforms: |
+            linux/amd64
+            linux/arm64
+          file: Dockerfile
+          tags: ghcr.io/goharbor/harbor-operator:latest
```
Comment (MarcelMue) [inline .github/workflows/release.yml:84]: Missing newline

File: Makefile [hunk 5]
```diff
@@ -209,10 +209,13 @@ go.sum: go.mod $(GONOGENERATED_SOURCES)
...
 # Build the docker image
 .PHONY: docker-build
-docker-build: dist/harbor-operator_linux_amd64/manager
-	docker build dist/harbor-operator_linux_amd64 \
-		-f Dockerfile \
-		-t "$(IMG)"
+docker-build:
+	docker buildx build \
+    		--platform linux/arm64,linux/amd64 \
+    		--push \
```
Comment (MarcelMue) [inline Makefile:215]: Why does build now also push?

Comment (MarcelMue) [reply Makefile:215]: It seems like this is failing CI

Comment (lengrongfu) [reply Makefile:221]: add `make docker-build-mutil-arch` cmd, reserve `make docker-build`.

File: .github/workflows/release.yml (hunk 4 above)
Comment (lengrongfu) [reply .github/workflows/release.yml:84]: What do you expect?

File: Makefile [hunk 6]
```diff
@@ -214,6 +214,15 @@ docker-build: dist/harbor-operator_linux_amd64/manager
 		-f Dockerfile \
 		-t "$(IMG)"
 
+.PHONY: docker-build-mutil-arch
+docker-build-mutil-arch:
```
Comment (thcdrt) [inline Makefile:218]:
```suggestion
docker-build-multi-arch:
```

**Comment by lengrongfu** [comment]
@wy65701436 Hi, PTAL.

**Comment by MarcelMue** [comment]
Can you update the branch to be based on `master`? I think CI should go green then.

**Review by MarcelMue — APPROVED** [review]

**Pull request merged**